### PR TITLE
Helmerator: initial commit

### PR DESCRIPTION
**Pull request by ozdanborne**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 31bb1cae73627538e223ee9ec376c6eda00de370

https://deploy-preview-3349--calico.netlify.com

File: _includes/charts/tigera-operator/templates/crds/01-crd-installation.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: installations.operator.tigera.io
+  annotations:
```
Comment (tmjd) [inline _includes/charts/tigera-operator/templates/crds/01-crd-installation.yaml]: Has this been added to the operator (and the changes below)? We update this CRD with the one that gets generated from the operator when needed.

Comment (ozdanborne) [reply _includes/charts/tigera-operator/templates/crds/01-crd-installation.yaml]: It sure has! https://github.com/tigera/operator/commit/3ddcbc5965019322f20bb771e54334b492bd57d7

Comment (tmjd) [reply _includes/charts/tigera-operator/templates/crds/01-crd-installation.yaml]: That change has the changes below (to the description and enum). I was also concerned with this annotation too which I don't see in that PR and looking at master doesn't exist.

**Pull request merged**